lms/templates/courseware/gradebook.html

```diff
@@ -118,7 +118,17 @@
         </tbody>
       </table>
     </div>
+    <span class="grade-book-footer">
+        %if page["previous_offset"] is not None:
+          <a href="${page_url}?offset=${page['previous_offset']}">&#12296;</a>
```

Comment:
For lack of a better example, can we use the same pattern lms uses for courseware navigation? Something like:

```
<button class="sequence-nav-button button-next">
        <span class="icon fa fa-chevron-next" aria-hidden="true"></span><span class="sr">Next</span>
    </button>
```